Ranger crashes on startup

#### Runtime Environment
<!-- Retrieve Python/ranger version and locale with `ranger --version` -->
- Operating system and version: MacOS 
- Terminal emulator and version: Iterm2 3.3
- Python version: Python version: 2.7.10 (default, Feb 22 2019, 21:55:15) [GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.37.14)]
- Ranger version/commit: 1.9.2
- Locale: None.None

#### Current Behavior
Intermittent crashes on startup

#### Expected Behavior
App works as normal

#### Steps to reproduce
1. Occasionally the app crashes on startup, unable to reproduce consistently. 

#### Traceback
<!-- If ranger crashes, paste the traceback in the quotes below. -->
```
ranger version: ranger 1.9.2
Python version: 2.7.10 (default, Feb 22 2019, 21:55:15) [GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.37.14)]
Locale: None.None

Traceback (most recent call last):
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/core/main.py", line 195, in main
    fm.loop()
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/core/fm.py", line 394, in loop
    ui.redraw()
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/gui/ui.py", line 339, in redraw
    self.finalize()
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/gui/ui.py", line 394, in finalize
    DisplayableContainer.finalize(self)
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/gui/displayable.py", line 264, in finalize
    displayable.finalize()
  File "/usr/local/Cellar/ranger/1.9.2/libexec/ranger/gui/widgets/view_base.py", line 60, in finalize
    - self.main_column.scroll_begin
AttributeError: 'NoneType' object has no attribute 'pointer'
```

I've got the same problem. 

System:
Manjaro
Python 3.7
Terminator 1,9-6

```
 19:43:07  ~  ranger   master 
 mcamp  Redhorse  $  ranger
ranger version: ranger-master 
Python version: 3.7.4 (default, Jul 16 2019, 07:12:58) [GCC 9.1.0]
Locale: en_US.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 171, in main
    fm.initialize()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 131, in initialize
    self.bookmarks.load()
  File "/usr/lib/python3.7/site-packages/ranger/container/bookmarks.py", line 47, in load
    new_dict = self._load_dict()
  File "/usr/lib/python3.7/site-packages/ranger/container/bookmarks.py", line 237, in _load_dict
    dct[key] = self.bookmarktype(value)
  File "/usr/lib/python3.7/site-packages/ranger/container/directory.py", line 150, in __init__
    assert not os.path.isfile(path), "No directory given!"
AssertionError: No directory given!

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

running `ranger --clean` does launch ranger. I have tried reinstalling and that didn't fix it.

--- a/ranger/core/tab.py
+++ b/ranger/core/tab.py
@@ -4,7 +4,7 @@
 from __future__ import (absolute_import, division, print_function)
 
 import os
-from os.path import abspath, normpath, join, expanduser, isdir
+from os.path import abspath, normpath, join, expanduser, isdir, dirname
 import sys
 
 from ranger.container import settings
@@ -123,9 +123,11 @@ class Tab(FileManagerAware, SettingsAware):  # pylint: disable=too-many-instance
 
         # get the absolute path
         path = normpath(join(self.path, expanduser(path)))
+        selectfile = None
 
         if not isdir(path):
-            return False
+            selectfile = path
+            path = dirname(path)
         new_thisdir = self.fm.get_directory(path)
 
         try:
@@ -155,6 +157,8 @@ class Tab(FileManagerAware, SettingsAware):  # pylint: disable=too-many-instance
         self.thisdir.sort_directories_first = self.fm.settings.sort_directories_first
         self.thisdir.sort_reverse = self.fm.settings.sort_reverse
         self.thisdir.sort_if_outdated()
+        if selectfile:
+            self.thisdir.move_to_obj(selectfile)
         if previous and previous.path != path:
             self.thisfile = self.thisdir.pointed_obj
         else:

